### DISCUSSION TITLE
code generate

### DISCUSSION THREAD

**Discussion by imlixixi** [nasa/fprime#1321] [Q&A]
In F Prime, I want to automatically generate algorithm model code(just like scade/Simulink). So I want to know whether the code algorithm model generated by scape / Simulink can be encapsulated into F‘s components ？

**Comment by LeStarch** [comment]
I haven't used those tools, but if they generate C or C++ then it should integrate with F´.  You'd just build a wrapper component around the output. Does this answer your question?
